core/src/components/spinner/test/resize/spinner.e2e.ts

```diff
@@ -0,0 +1,20 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+
+/**
+ * This behavior does not vary across directions.
+ */
+configs({ directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
```

Comment:
Spinner has no per-mode styles, so we don't need both iOS and MD screenshots.

Reply:
Yeah you can, though spinner does have per-mode _behavior_, so we should probably have a spec test that checks that the default spinner is set correctly on each mode: https://github.com/ionic-team/ionic-framework/blob/eba4c7f6e629dedf68de24a512e92b7ce4c43154/core/src/components/spinner/spinner.tsx#L42-L48

We might be able to check that the correct `spinner-*` class is set. 

I'd also be fine if we made a ticket and updated the non-resize tests as a separate task.